### DISCUSSION TITLE
Paypal donations impossible, please consider BTC donations.

### DISCUSSION THREAD

**Discussion by lanmaster** [onemen/TabMixPlus#302] [Ideas]
Paypal donations impossible, please consider BTC donations.

**Comment by pyrates999** [comment]
since bitcoin is not a currency, I doubt it.

**Comment by Surfit1111** [comment]
Donating from Japan is not supported, they say.

**Comment by MememeOyaji** [comment]
I can not donate from Japan.
Perhaps due to PayPal system.
